tasks/libs/releasing/version.py

```diff
@@ -256,7 +256,10 @@ def get_version(
 ):
     version = ""
     if pipeline_id is None:
-        pipeline_id = os.getenv("CI_PIPELINE_ID")
+        if os.getenv("E2E_PIPELINE_ID"):  # If we are in an E2E pipeline, we should use the E2E pipeline ID
+            pipeline_id = os.getenv("E2E_PIPELINE_ID")
+        else:
+            pipeline_id = os.getenv("CI_PIPELINE_ID")
```

Comment:
I'm not sure which one is the more readable but here is another proposition:
```suggestion
        pipeline_id = os.getenv("E2E_PIPELINE_ID", os.getenv("CI_PIPELINE_ID"))  # If we are in an E2E pipeline, we should use the E2E pipeline ID
```